bot/seasons/halloween/hacktoberstats.py

```diff
@@ -10,16 +10,18 @@
 import discord
 from discord.ext import commands
 
+log = logging.getLogger(__name__)
 
-class Stats:
+
+class HacktoberStats:
     def __init__(self, bot):
         self.bot = bot
-        self.link_json = Path('./bot/resources', 'github_links.json')
+        self.link_json = Path("bot", "/resources", "github_links.json")
```

Comment:
I don't really understand how Path works at all but this feels wrong.
```py
>>> Path("bot", "resources", "github_links.json")
PosixPath('bot/resources/github_links.json')
>>> Path("bot", "/resources", "github_links.json")
PosixPath('/resources/github_links.json')
```
This looks like there is a `resources` dir at root, which I'm not aware of.

Reply:
It's a remnant of when it was on the main bot, which has does have a [resources directory](https://github.com/python-discord/bot/tree/master/bot/resources).